frontend/src/router/routeHandlers.ts

```diff
@@ -30,24 +32,33 @@ import type { RouteLocationNormalized } from 'vue-router';
 
 const ACCESS_RESTRICTED_ERROR = new FamRouteError(
     RouteErrorName.ACCESS_RESTRICTED,
-    "Access restricted"
+    'Access restricted'
 );
 
 // --- beforeEnter Route Handler
 
 const beforeEnterDashboardRoute = async (to: RouteLocationNormalized) => {
     let applicationAdmins;
     let userRolesFetchResult;
+    let delegatedAdmins;
+
     if (selectedApplicationId.value === FAM_APPLICATION_ID) {
-        applicationAdmins = await asyncWrap(fetchApplicationAdmins())
+        applicationAdmins = await asyncWrap(fetchApplicationAdmins());
     } else {
         userRolesFetchResult = await asyncWrap(
             fetchUserRoleAssignments(selectedApplicationId.value)
         );
+
+        if (LoginUserState.isApplicationAdmin()) {
+            delegatedAdmins = await asyncWrap(
+                fetchDelegatedAdmins(selectedApplicationId.value)
+            );
+        }
```

Comment:
Like Catherine mentioned, better rename the `isApplicationAdmin`.
Also probably can even move the simple `if (LoginUserState.isApplicationAdmin())` statement  inside `fetchDelegatedAdmin` to make routeHandlers less confusing.

Reply:
Changed the name and moved it to the fetchDelegatedAdmin func

Reply:
oh I just saw Ian's comment.... okay, then let's keep it for now, I'll chat with him later. thanks!!